dotcom-rendering/src/web/components/DynamicSlowMPU.tsx

```diff
@@ -29,57 +29,58 @@ const Card33_ColumnOfThree33_Ad33 = ({
 	containerPalette?: DCRContainerPalette;
 	showAge?: boolean;
 	index: number;
-}) => (
-	<UL direction="row">
-		<LI percentage="33.333%" padSides={true}>
-			<FrontCard
-				trail={cards[0]}
-				containerPalette={containerPalette}
-				showAge={showAge}
-			/>
-		</LI>
-		<LI
-			percentage="33.333%"
-			showDivider={true}
-			showTopMarginWhenStacked={true}
-		>
-			<UL direction="column">
-				<LI padSides={true} padBottom={true}>
-					<FrontCard
-						trail={cards[1]}
-						containerPalette={containerPalette}
-						showAge={showAge}
-						imageUrl={undefined}
-						headlineSize="small"
-					/>
-				</LI>
-				<LI padSides={true} padBottom={true}>
-					<FrontCard
-						trail={cards[2]}
-						containerPalette={containerPalette}
-						showAge={showAge}
-						imageUrl={undefined}
-						headlineSize="small"
-					/>
-				</LI>
-				<LI padSides={true}>
-					<FrontCard
-						trail={cards[3]}
-						containerPalette={containerPalette}
-						showAge={showAge}
-						imageUrl={undefined}
-						headlineSize="small"
-					/>
-				</LI>
-			</UL>
-		</LI>
-		<LI percentage="33.333%">
-			<Hide until="tablet">
-				<AdSlot position="inline" index={index} />
-			</Hide>
-		</LI>
-	</UL>
-);
+}) =>
+	cards[0] && cards[1] && cards[2] && cards[3] ? (
```

Comment:
We make sure the cards exist.

An alternative would be to type the cards array as a tuple of 4 items, and handle errors one level above.